Formula/go-jira.rb

```diff
@@ -0,0 +1,23 @@
+class GoJira < Formula
+  desc "Simple jira command-line client in Go"
+  homepage "https://github.com/Netflix-Skunkworks/go-jira"
+  url "https://github.com/Netflix-Skunkworks/go-jira/archive/v1.0.12.tar.gz"
+  sha256 "7ce5d6dcc5021f2019b0107c4ad342979d18bb110188b810017b2d94ff384a33"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    dir = buildpath/"src/gopkg.in/Netflix-Skunkworks/go-jira.v1"
+    dir.install buildpath.children
+
+    ldflags = ["-w", "-s"]
+    cd dir do
+      system "go", "build", "-o", bin/"jira", "-ldflags", ldflags.join(" "), "cmd/jira/main.go"
+    end
+  end
+
+  test do
+    system "#{bin}/jira", "export-templates"
```

Comment:
Does this produce any output/files we can test the contents of?

Reply:
is the existence check enough or would you like a contents check too? The expected content string(s) could be rather large, so (to me) verifying that the command dumped out files to the right location is enough to show that it compiled correctly

Reply:
@woodruffw Does this work for your purposes? This command, `version`, and `help` are the only ones that don't require an Atlassian Cloud instance to test.

I also added a test for the contents of the smallest exported template I could find